modules/op-setting-eventlistener-scc.adoc

```diff
@@ -0,0 +1,47 @@
+// This module is included in the following assemblies:
+// * secure/securing-webhooks-with-event-listeners.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='op-setting-eventlistener-scc_{context}']
+= Configuring security context for event listeners
+
+You can configure a custom security context directly in your `EventListener` definition to ensure that containers run as a non-root user and with restricted access to the file system. 
```

Comment:
@ochromy can we update this to say that 
```
You can configure a custom security context directly in your `EventListener` definition based on what securityContext user wants to set  ex: to run container as non run now user can set runAsNonRoot: true
``` 
something like this

My point is lets not mention specific field of securityContext